troposphere/cloudfront.py

```diff
@@ -111,6 +120,7 @@ class Restrictions(AWSProperty):
 
 class ViewerCertificate(AWSProperty):
     props = {
+        'AcmCertificateArn': (basestring, False),
```

Comment:
It would be nice to have a test that checks that only one of `AcmCertificateArn`, `CloudFrontDefaultCertificate`, or `IamCertificateId`  is used.